### PR TITLE
Pinecone samples: PebbloSafeLoader and PebbloSafeRetrievalQA with Identity and Semantic Enforcement

### PR DESCRIPTION
**Pull request by Raj725**
Added samples for PebbloSafeLoader and PebbloSafeRetrievalQA with Identity and Semantic Enforcement using Pinecone.

Logs for Safeloader:

```
Loaded 119 documents ...


Semantic Stats:
Authorized Identities: ['demo-hr-group@daxa.ai', 'demo-user-hr@daxa.ai', 'demo-eng-group@daxa.ai']
Semantic Topics: ['employee-agreement', 'sexual-harassment', 'patent-application-fillings', 'financial-report']
Semantic Entities: []


Deleting and recreating index: identity-semantic-enforcement-rag ...
Deleted index: identity-semantic-enforcement-rag.
Creating index: identity-semantic-enforcement-rag...
Loading docs into index...
Waiting for index to be ready...
Done!
First document: page_content='\ufeffConsolidated Performance Repor.......'
Finished hydrating Vector DB ...
```
 
**Upcoming PR: Move duplicate code to common place**

### CODE REVIEW

File: pebblo/topic_classifier/topic_classifier.py [hunk 1]
```diff
@@ -85,6 +85,6 @@ def _get_topics(topic_model_response):
 
         final_topic = {}
         if len(topics) > 0:
-            most_possible_advice = max(topics, key=topics.get)
+            most_possible_advice = max(topics, key=lambda t: topics[t])
```
[the remark this answers is not in the document]
Comment (Raj725) [reply pebblo/topic_classifier/topic_classifier.py]: Removed

File: pebblo_safeloader/langchain/identity-rag-pinecone/.env.sample [hunk 2]
```diff
@@ -0,0 +1,5 @@
+PEBBLO_CLASSIFIER_URL="http://localhost:8000/"
```
[the remark this answers is not in the document]
Comment (Raj725) [reply pebblo_safeloader/langchain/identity-rag-pinecone/.env.sample]: Is there any issue with including it? BTW, replaced it with URL syntax.

File: pebblo_safeloader/langchain/identity-rag-pinecone/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+## PebbloSafeLoader sample app with Identity and Semantic metadata
+
+This solution uses the following PR:
+
+1. enable prompt and dscover apis for PebbloSafeRetriever. #31
```
[the remark this answers is not in the document]
Comment (Raj725) [reply pebblo_safeloader/langchain/identity-rag-pinecone/README.md]: That was the PR title. It's not needed anymore

File: pebblo_safeloader/langchain/identity-rag-pinecone/.env.sample (hunk 2 above)
[the remark this answers is not in the document]
Comment (shreyas-damle) [reply pebblo_safeloader/langchain/identity-rag-pinecone/.env.sample]: We can include it for better clarity.

File: pebblo_safeloader/langchain/identity-rag-pinecone/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- _folder_id_: Google Drive folder ID where the documents are stored
+
+
+5. Run langchain sample app PebbloSafeLoader and PebbloRetrievalQA
+
+```console
+$ python3 pebblo_safeload.py
+```
+
+6. Retrieve the Pebblo PDF report in `$HOME/.pebblo/pebblo-loader-sematic-rag-pinecone/pebblo_report.pdf` file path on the system
```
[the remark this answers is not in the document]
Comment (Raj725) [reply pebblo_safeloader/langchain/identity-rag-pinecone/README.md]: Added

File: pebblo_saferetriever/langchain/identity-n-semantic-rag-pinecone/data_load.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+def create_pinecone_index(pinecone_index_name: str, recreate: bool = False):
+    """
+    Create a new Pinecone index
+    """
+
+    # configure client
+    pc = Pinecone(api_key=PINECONE_API_KEY)
+
+    use_serverless = False
```
[the remark this answers is not in the document]
Comment (Raj725) [reply pebblo_saferetriever/langchain/identity-n-semantic-rag-pinecone/data_load.py]: Keeping else part only

**Comment by Raj725** [comment]
Closing this PR. I will create a new PR with the updated folder structure.

**Pull request closed without merging**